tests/modules/nf-core/macrel/contigs/test.yml

```diff
@@ -1,16 +1,15 @@
 - name: macrel contigs test_macrel_contigs
   command: nextflow run ./tests/modules/nf-core/macrel/contigs -entry test_macrel_contigs -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/macrel/contigs/nextflow.config
   tags:
-    - macrel/contigs
     - macrel
+    - macrel/contigs
   files:
     - path: output/macrel/test/README.md
       md5sum: fa3706dfc95d0538a52c4d0d824be5fb
     - path: output/macrel/test/test.all_orfs.faa.gz
     - path: output/macrel/test/test.prediction.gz
     - path: output/macrel/test/test.smorfs.faa.gz
-      md5sum: 79704c6120c2f794518301af6f9b963d
+      md5sum: ba1d2339474bc393e6e4c7f28ef4fd46
     - path: output/macrel/test/test_log.txt
-      md5sum: 6fdba143dce759597eb9f80e5d968729
     - path: output/macrel/versions.yml
-      md5sum: be8bf0d0647751c635c3736655f29f85
+      contains: ["macrel"]
```

Comment:
That doesn't look right - the version should be just numbers: 1.2.0 in this case?

Normally we just check for the presence of this  file, not the contents

Reply:
ok in that case  i removed the md5sums and added `contains: ["macrel"]`

Reply:
You will? Or Have/ I don't see any change at th moment..

Reply:
You will? Or Have/ I don't see any change at th moment..

Reply:
Now, it didnt push for some reason